week_4/project/week_4.py

```diff
@@ -20,7 +20,9 @@ def get_s3_data(context, group_name="corise"):
     return output
 
 
-@asset()
+@asset(
+    group_name="corise"
+)
 def process_data(get_s3_data):
     high_val = 0
```

Comment:
@dfg129 I find this is very helpful - https://www.geeksforgeeks.org/python-get-the-object-with-the-max-attribute-value-in-a-list-of-objects/